lib/utils/remove-accents.ts

```diff
@@ -0,0 +1,11 @@
+/**
+ * Returns a string with accents stripped
+ *
+ * @param {String} inputString string for which to remove accents
+ * @returns {String} string with accents removed
+ */
+const removeAccents = (content: string): string => {
+  return content.normalize('NFD').replace(/[\u0300-\u036f]/g, '');
```

Comment:
this is a good start here but have a lot more to do. for instance, `ø` is code point `U+00F8` and things like Hebrew vowel pointing and Japanese dukuten/handakuten should also go. sadly I was unable to find a satisfactory method native to JS to do this. `Intl.Collator` provides an appropriate string comparator but unfortunately doesn't provide the stripped text or provide string matching.

that means we're probably limited practically to _getting closer_ to support than _supporting_ localized search, but we can hit some of those common groups here at the same time. this would be a good place for unit tests that show the extent of our support as well as places we know we aren't in compliance.

might want to cite copied code as well 👍